src/content/changelog/4-8-0-2019-10-21.md

```diff
@@ -0,0 +1,25 @@
+---
+title: '4.8.0: 2019-10-21'
```

Comment:
This should be 4.7

Reply:
@yoshizzle the current spec is at 4.7.0, and this update added the object-storage/cancel endpoint. May I ask why the new release would also be 4.7.0?

Reply:
oh wait, actually nevermind. I had my repos mixed up.

Nothing to see here, sorry!

Reply:
🙏